freqtrade/rpc/api_server/ws/channel.py

```diff
@@ -95,6 +105,20 @@ def subscribed_to(self, message_type: str) -> bool:
         """
         return message_type in self._subscriptions
 
+    async def relay(self):
+        """
+        Relay messages from the channel's queue and send them out. This is started
+        as a task.
+        """
+        while True:
+            message = await self.queue.get()
+            try:
+                await self._send(message)
+                self.queue.task_done()
+            except RuntimeError:
```

Comment:
would it make sense to move the "sleep" here - so we can send unlimited messages from the main queue, and are eventually limiting performance here?

Reply:
I agree, I will make this change